R/Sound_DMR.R

```diff
@@ -1116,31 +1121,33 @@ sound_score <- function(changepoint_OF = dataframe, Statistic="Z_GroupT_small",
 #' @param Methyl_bed (df) - Data frame containing the ONT methylation calls in a bed file for each individual separately
 #' @param Sample_ID (str) - A string that is used inplace of sample name to keep it uniform. We are using an enumerator to generate this based on the number of samples/individuals in the experiment.
 #' @param Methyl_call_type (str) - A string that includes information about the type of run. Currently this package works on Megalodon , DSP (DeepSignal Plant), Dorado and Bonito. Default call type is Dorado.
+#' @param max_read_depth (int) - This parameter can be used to filter out rows with high red depth such as highly repetative regions to not be included in the analysis. Default is 0. 
 #' @return Methyl_bed_sub (df) - Standard data frame methyl file for every individual with Meth, Unmeth and Per_Meth columns
 #' @import tidyverse
 #' @import stringr
 #' @export
 
-get_standard_methyl_bed <-function(Methyl_bed="Methyl.bed", Sample_ID = "S1", Methyl_call_type="Dorado") {
-
+get_standard_methyl_bed <-function(Methyl_bed="Methyl.bed", Sample_ID = "S1", Methyl_call_type="Dorado", max_read_depth='') {
```

Comment:
I think we would want the default max read depth here to be an integer, otherwise I think you might get an error when running line 1147 below

Reply:
Good call! I'll actaully add default as 0 and make it so that if no one enters anything, we will not filter anything out..